tensorboard/plugins/graph/tf_graph_node_search/tf-graph-node-search.ts

```diff
@@ -30,7 +30,7 @@ class TfGraphNodeSearch extends LegacyElementMixin(PolymerElement) {
     <div id="search-container">
       <paper-input
         id="runs-regex"
-        label="Search nodes. Regexes supported."
+        label="Search nodes by regex"
```

Comment:
We currently have three copies for this.

![image](https://user-images.githubusercontent.com/2547313/112023171-39b6d680-8af0-11eb-980e-408e672fa1c8.png)

and 

![image](https://user-images.githubusercontent.com/2547313/112023188-3facb780-8af0-11eb-8440-430f0ae619d2.png)


How about `Search nodes (regex)`?

Reply:
Done, thanks.